<ui version="4.0" >
 <class>KPrPresentationTool</class>
 <widget class="QWidget" name="KPrConfigurePresenterView" >
  <property name="geometry" >
   <rect>
    <x>1</x>
    <y>1</y>
    <width>1</width>
    <height>1</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QToolButton" name="penButton">
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="highLightButton" >
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
    <item>
      <widget class="QToolButton" name="blackButton" >
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
